docs/reference/ingest/search-ingest-pipelines.asciidoc

```diff
@@ -0,0 +1,313 @@
+[[ingest-pipeline-search]]
+== Ingest pipelines in Search
+
+<<ingest, Ingest Pipelines>> are an Elasticsearch native tool which let you perform common transformations on data before storing it in Elasticsearch.
```

Comment:
Because this used to live in the Enterprise Search docs, the introduction provides context for ingest pipelines. Now that this doc is within the ingest pipelines documentation, we may be able to assume that context and instead provide context on the Content UI.

For example, we could communicate:

- You can manage ingest pipelines through Elasticsearch APIs or Kibana UIs
- The Content UI is for creating and managing indices optimized for search use cases (non time series data)
- When managing indices through that UI, you may prefer to manage ingest pipelines there
- This doc tells you how to do that